packages/client/tests/functional/issues/9007/tests.ts

```diff
@@ -24,17 +22,7 @@ testMatrix.setupTestSuite(
       ).rejects.toThrowError()
     })
 
-    test('should not generate the contains field on the where type', async () => {
-      const generatedTypeScriptSrc = await fs.promises.readFile(
```

Comment:
This part of the test was added before. What it does:
It checks that a generated type does not have a `contains` property.